modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/xeno.dm

```diff
@@ -4,6 +4,7 @@
 	id = SPECIES_XENO
 	say_mod = "hisses"
 	default_color = "#00FF00"
+	family_heirlooms = list(/obj/item/toy/plush/rouny, /obj/item/toy/toy_xeno,)
```

Comment:
Hmmm, not 100% sure about the toy xeno (mostly for mechanics reason), the rouny is cute though :)
Still think it's fine regardless, it's really not too bad.

Reply:
I understand. Though considering the amount of xeno players (I've seen two at most.) I think it should be fine.